main.tf

```diff
@@ -3680,3 +3680,117 @@ module "aws_gateway_api_controller" {
 
   tags = var.tags
 }
+
+################################################################################
+# Bottlerocket Update Operator
+################################################################################
+locals {
+  wait_for_cert_manager = var.cert_manager.wait ? [module.cert_manager] : []
+}
+
+module "bottlerocket_update_operator_crds" {
```

Comment:
I thought to keep the name bottlerocket-shadow as it's the chart name, but at the same time, it doesn't mean anything, and in the [BRUPOP doc](https://github.com/bottlerocket-os/bottlerocket-update-operator/tree/develop/deploy/charts/bottlerocket-shadow), they run a `helm install brupop-crd deploy/charts/bottlerocket-shadow` to deploy this one.  Can change the module to `bottlerocket-shadow` for consistency the name as `-crds` or something more meaningful?

Reply:
>Bottlerocket shadows are the "reflections" of the bottlerocket nodes themselves.

Pretty legit :)

Your point is a valid one. However, this module is only invoked when `enable_bottlerocket_update_operator = true`, i.e. users are not going to be directly enabling. I am really hesitant to make an exception because it leads us down a slippery slope of diverging away from upstream which we are trying to avoid as a blanket rule.

Reply:
Both ways work for me, I just want to make sure we point customers to some meaningful parameter if they need to customize anything, and even with the `enable_bottlerocket_update_operator = true`, the parameters to change the will/can reside in `bottlerocket_update_operator_crds = {}`.

You think is a bad idea to do something like this?

```
module "bottlerocket_shadow" {
  name          = try(var.bottlerocket_update_operator_crds.name, "bottlerocket-update-operator-crds")
}
```

I mean, name the module as `bottlerocket-shadow` but the Helm Chart as `bottlerocket_update_operator_crds`, same approach they suggest in the docs.